src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -213,3 +213,50 @@ export const createStandardChangeRequest = async (req: Request, res: Response) =
     message: `Successfully created standard change request #${createdChangeRequest.crId}.`
   });
 };
+
+export const addProposedSolution = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({ errors: errors.array() });
+  }
+
+  const { body } = req;
+
+  // verify user is allowed to create stage gate change requests
+  const user = await prisma.user.findUnique({ where: { userId: body.submitterId } });
+  if (!user) {
+    return res.status(404).json({ message: `user with id #${body.submitterId} not found` });
+  }
+  if (user.role === Role.GUEST) return res.status(401).json({ message: 'Access Denied' });
+
+  // ensure existence of scope change request
+  const foundScopeCR = await prisma.scope_CR.findUnique({ where: { scopeCrId: body.scopeCrId } });
+  if (!foundScopeCR)
+    return res
+      .status(404)
+      .json({ message: `scope change request with id #${body.scopeCrId} not found` });
+
+  // ensure existence of change request
+  const foundCR = await prisma.change_Request.findUnique({
```

Comment:
ok this is my bad because i told you to use the scopecrId but actually the frontend won't send a scopeCRId at all. If you look at change-request.types.ts in the shared module, you can see that only the base crId exists. So the frontend will send just the changerequestid and then you will have to find a scope cr that has that id connected to it.